network_simulator/common.py

```diff
@@ -25,6 +25,11 @@
 """Initial router packet id of :class:`.RouterPacket` is -1."""
 ROUTER_PACKET_DEFAULT_ID = -1
 
+"""Conversion to/from bits <-> KB, MB"""
+KILOBYTE = 8 * 1024
```

Comment:
Use floats whenever possible (8.0 vs 8) to make sure there's no nasty integer division bugs later.